### PR TITLE
Localization

### PR DESCRIPTION
**Pull request by hduong1207uet**
## Related Tickets:
- [#35050](https://edu-redmine.sun-asterisk.vn/issues/35050)

## WHAT this PR does?

- localiization function

## Notes Impacted Areas:
- N/A.

## Attachments:
- image
- image

### CODE REVIEW

**Comment by hduong1207uet** [comment]
ready

File: resources/lang/en.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "Login" : "Login",
+    "Register" : "Register",
+    "Dashboard" : "Dashboard",
+    "You_are_logged_in!" : "You are logged in!",
+    "E-Mail_address" : "E-Mail Address",
+    "Password" : "Password",
+    "Remember_me" : "Remember Me",
+    "Forgot_password?" : "Forgot Password?",
```
Comment (ghost) [inline resources/lang/en.json]:
```suggestion
    "Forgot_password" : "Forgot Password ?",
```

File: resources/lang/vi.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "Login" : "Đăng nhập",
+    "Register" : "Đăng ký",
+    "Dashboard" : "Trang chính",
+    "You_are_logged_in!" : "Bạn đã đăng nhập thành công !",
+    "E-Mail_address" : "Địa chỉ Email",
+    "Password" : "Mật khẩu",
+    "Remember_me" : "Nhớ tôi trên thiết bị",
+    "Forgot_password?" : "Quên mật khẩu ?",
```
Comment (ghost) [inline resources/lang/vi.json]:
```suggestion
    "Forgot_password" : "Quên mật khẩu ?",
```

File: resources/views/auth/login.blade.php [hunk 3]
```diff
@@ -59,7 +59,7 @@
 
                                 @if (Route::has('password.request'))
                                     <a class="btn btn-link" href="{{ route('password.request') }}">
-                                        {{ __('Forgot_password') }}
+                                        {{ __('Forgot_password?') }}
```
Comment (ghost) [inline resources/views/auth/login.blade.php]:
```suggestion
                                        {{ __('Forgot_password') }}
```

**Comment by hduong1207uet** [comment]
ready

**Review by ghost — APPROVED** [review]

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**